server/public/styles.css

```diff
@@ -0,0 +1,80 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: auto 0;
+}
+
+i {
+  font-size: 1.25rem;
+  filter: drop-shadow(3px 3px 2px grey);
+}
+
+img {
+  object-fit: contain;
+  max-height: 10rem;
+}
+
+.title {
+  text-align: center;
+}
+
+.title > h1 {
+  background: radial-gradient(50% 50% at 50% 50%, #c5f9fa 25%, rgba(255, 255, 255, 0) 100%), #66acad;
+  font-family: 'Almendra SC', serif;
+  font-size: 2rem;
+  padding: 1rem;
+  margin: auto;
+}
+
+i:hover,
+h3:hover,
+h4:hover {
+  cursor: pointer;
+}
+
+.container.nav {
+  display: flex;
+  justify-content: space-around;
+  padding: 1rem;
+  background-color: #c5f9fa;
+}
+
+.search {
+  margin-right: 1rem;
+  padding-left: 0.25rem;
+  border-radius: 50px;
+  border: 0 white solid;
+}
```

Comment:
Double check that all of this CSS is doing something for you. Also, check Bootstrap's spacing, flex, and border helpers.

https://getbootstrap.com/docs/4.5/utilities/borders/
https://getbootstrap.com/docs/4.5/utilities/flex/
https://getbootstrap.com/docs/4.5/utilities/spacing/